flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/sql/GroupWindowTest.scala

```diff
@@ -161,7 +161,7 @@ class GroupWindowTest extends TableTestBase {
             "rowtime('w$) AS w$rowtime",
             "proctime('w$) AS w$proctime")
         ),
-        term("select", "EXPR$0", "DATETIME_PLUS(w$end, 60000) AS EXPR$1")
+        term("select", "EXPR$0", "+(w$end, 60000) AS EXPR$1")
```

Comment:
Do you know the reason for this change? Why is `DATETIME_PLUS` now equivalent to `+`? It is not deprecated. Can we replace all occurrences of this constant in our code base?

Reply:
@twalthr code relevant is around `CodeGenerator#L769` and I think `+` is equivalent to `DATETIME_PLUS` here and below.
However, I see no reason for this change. Not only `DATETIME_PLUS` works as expected, but also `DATETIME_PLUS` is more clear than bare `+`. How can one know `(w$end, 60000)` is about `DATETIME` without take a near look.

Reply:
FYI, code around `CodeGenerator#L769` like:
```scala
case PLUS | DATETIME_PLUS if isTemporal(resultType) =>
        val left = operands.head
        val right = operands(1)
        requireTemporal(left)
        requireTemporal(right)
        generateTemporalPlusMinus(plus = true, nullCheck, left, right, config)
```

Reply:
Thanks for the research. @suez1224 if you find time, you could discuss this in the Calcite community again. I find it a bit confusing to have the same same string for different SQL operators. But it is not a blocker for this PR.